### ISSUE TITLE
[feature]Is it possible to work in the Android terminal?  

### ISSUE THREAD

**Issue by huazhaozhe** [skylot/jadx#1229]
*Describe your idea:*
For example
https://play.google.com/store/apps/details?id=jackpal.androidterm
thanks

**Comment by jpstotz** [comment]
Command-line version of jadx should work if you install Java 8+ inside your terminal environment before starting jadx. 

An Android based GUI version is a bit out of scope of this project.
